codex/raw/marketplace.md

```diff
@@ -0,0 +1,269 @@
+---
+title: CODEX-MARKETPLACE
+name: Codex Storage Marketplace
+status: raw
+tags: codex
+editor: Dmitriy <dryajov@status.im>
+contributors:
+- Mark <mark@codex.storage>
+- Adam <adam.u@status.im>
+- Eric <ericmastro@status.im>
+- Jimmy Debe <jimmy@status.im>
+---
+
+## Abstract
+
+This specification describes a method for Codex storage providers and client nodes to participate in a storage marketplace. 
+The goal is to create a storage marketplace that promotes durability.
+
+## Motivation
+The Codex network aims to create a peer-to-peer storage engine with strong data durability, 
+data persistence guarantees and node storage incentives.
+To reach this goal, a data availability and retrieval mechanism is needed.
```

Comment:
Not sure where this came from, Codex isn't about **data availability**, it is strictly about **data durability**.

Reply:
I found this in the codex white-paper, data availability is mentioned often. "This makes data availability one of the principal requirements for decentralized storage. To reach that goal, Codex implements strong incentives and a robust data dispersion scheme that allows for extreme flexibility at the moment of accessing a dataset."